daemon/graphdriver/driver.go

```diff
@@ -142,10 +142,36 @@ func GetDriver(name, home string, options []string) (Driver, error) {
 func New(root string, options []string) (driver Driver, err error) {
 	for _, name := range []string{os.Getenv("DOCKER_DRIVER"), DefaultDriver} {
 		if name != "" {
+			logrus.Infof("[graphdriver] trying provided driver %q", name) // so the logs show specified driver
 			return GetDriver(name, root, options)
 		}
 	}
 
+	// Guess for prior driver
```

Comment:
I mean for current users, this might already be a problem.  They might already be "auto" on devicemapper, for example, and had been kicked there from AUFS (but of course their images were gone so they went along their merry way and recreated everything in devicemapper), so they have data in their `/var/lib/docker` for both.